src/components/FilmstripViewer.jsx

```diff
@@ -18,6 +19,7 @@ class FilmstripViewer extends React.Component {
 
   changeFrame(i) {
     this.props.dispatch(changeFrame(i));
+    this.props.dispatch(changeFrameData(i));
```

Comment:
I assume that, without this call, when you change from Frame 1 to Frame 2, Frame 1's Previous User Annotations and/or Aggregated Annotations From Other Users would still show?

Reply:
Actually, I noticed with a change in frame, the previous annotations would just disappear. There was probably some line that returned `null` or something if the data frame didn't match the current subject viewer frame.